src/test/resources/projects/maven/singleModMavenMP/pom.xml

```diff
@@ -77,7 +77,7 @@
             <plugin>
                 <groupId>io.openliberty.tools</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
-                <version>3.8.2</version>
+                <version>3.10.3</version>
```

Comment:
In some test cases `liberty-maven-plugin` is version 3.7.1. We should think about using the same version for all the tests.

Reply:
@turkeylurkey So you mean to change all `liberty-maven-plugin` to the latest? `3.7.1` version is present in src/test/resources/files/smNLTRestProject/maven/pom.xml ivdeya and src/test/resources/projects/maven/jakarta-sample/pom.xml.

Reply:
Similarly with the gradle version in src/test/resources/files/smNLTRestProject/gradle/build.gradle. also?

Reply:
Yes